test/e2e/e2e_test.go

```diff
@@ -37,7 +42,7 @@ var _ = Describe("leaderWorkerSet e2e tests", func() {
 				GenerateName: "test-ns-",
 			},
 		}
-		Expect(k8sClient.Create(ctx, ns)).To(Succeed())
+		Eventually(k8sClient.Create(ctx, ns)).Should(Succeed())
```

Comment:
Use Eventually is ok, do you meet failure ever especially in Bofore?

Reply:
+1, probably remove this change